deployment/roles/deploy/tasks/deploy-secrets.yml

```diff
@@ -11,6 +11,6 @@
     validate_certs: "{{ validate_certs }}"
   with_template:
     - secret-sentry.yml.j2
-    - centos-secrets.yml.j2
+    - secret-centos.yml.j2
```

Comment:
I wouldn't do this, b/c breaks the consistency in this repo, where the type of a resource is indicated as a suffix to the base filenames.